pkg/products/monitoring/reconciler.go

```diff
@@ -461,6 +461,15 @@ func (r *Reconciler) reconcileScrapeConfigs(ctx context.Context, serverClient k8
 func (r *Reconciler) reconcileDashboards(ctx context.Context, serverClient k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
 
 	for _, dashboard := range r.Config.GetDashboards() {
```

Comment:
What do you think to a refactor here:
```
for _, dashboard := range r.Config.GetDashboards(installation.Spec.Type) {
```
and
```
managedAPIDashboards = string{}[
 "dashboard1",
}
managedDashboards = string{}[
 "dashboard10",
}
func (c *Config) GetDashboards(instType string) string[] {
   case "managed-api":
      return managedApiDashboards
   case "managed":
      return managedDashboards
   default:
      return managedDashboards
}
```
I'm concerned that as we increase the number of exceptions below, it could get difficult to maintain.

Reply:
Good idea. I've updated the logic as suggested.